src/ImageSequencer.js

```diff
@@ -197,6 +197,25 @@ ImageSequencer = function ImageSequencer(options) {
     else modulesdata = modules[name][1];
     return modulesdata;
   }
+
+  function toString(step_name,options){
+    if(step_name){
+      return `${step_name}(${Object.keys(options).map(key=>key + ':' + options[key]).join(',')})`
+    }
+    else{
+      return  copy(this.images.image1.steps).map(step=>{
+        let inputs = modulesInfo(step.options.name).inputs;
+        
+        if(!inputs)
+          inputs = {}
+        
+        for(let input in inputs){
+          inputs[input] = step.options[input];
+        }
+        return toString(step.options.name,inputs);
```

Comment:
is this recursive then? Are we assembling an object containing the configurations? At what point do we return a stringified version? Sorry, maybe this is just readability or perhaps you're not done here. But this is off to a great start!